src/brain/issueLabelHandler/followups.ts

```diff
@@ -234,12 +234,12 @@ export async function ensureOneWaitingForLabel({
   let timeToRespondBy;
   if (labelName === WAITING_FOR_PRODUCT_OWNER_LABEL) {
     timeToRespondBy =
-      (await calculateSLOViolationTriage(
```

Comment:
Kinda surprising that Typescript will let you await a function that's not a future. Woulda been nice if the compiler had caught this!

Reply:
Hmmm, upon further inspection I see that it is a future. How exactly does this work in practice then? Does calling the future without awaiting just trigger an async call? But then how does the variable get set?

Reply:
Nm - reading further still I see that you changed the definition too. Disregard :)

Reply:
It's calculation using the moment library to return a future timestamp. The functions themselves are not asynchronous anymore, so it's not needed. The reason why I had `await/async` there previously is because I had db operations somewhere baked in there, which are now removed.